test/unittests/studyentities/CandID_Test.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * This file contains unit test for the CandID value object.
+ *
+ * PHP Version 7
+ *
+ * @category Tests
+ * @package  StudyEntities
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+namespace LORIS\StudyEntitites\Candidate\Test;
+
+require_once __DIR__ . '/../../../vendor/autoload.php';
+
+use \PHPUnit\Framework\TestCase;
+use \LORIS\StudyEntities\Candidate\CandID;
+
+/**
+ * Unit test class for the CandID value object
+ *
+ * PHP Version 7
+ *
+ * @category Tests
+ * @package  StudyEntities
+ * @author   Xavier Lecours <xavier.lecours@mcin.ca>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+class CandID_Test extends TestCase
+{
+    public function testConstructorEmptyValue(): void
+    {
+        $this->expectException(\ArgumentCountError::class);
+        $candid = new CandID();
+    }
```

Comment:
Adding the strict typing that I suggested would make this test meaningless (and invalid?)